cmd/fission-cli/app/app.go

```diff
@@ -86,7 +86,7 @@ func App() *cobra.Command {
 	})
 
 	wrapper.SetFlags(rootCmd, flag.FlagSet{
-		Global: []flag.Flag{flag.GlobalServer, flag.GlobalVerbosity, flag.KubeContext},
+		Global: []flag.Flag{flag.GlobalServer, flag.GlobalVerbosity, flag.KubeContext, flag.PreCheckOnly},
```

Comment:
Just exploring, if we can avoid global flag since this flag is used in check command only ?